docs_src/microservices/device/Ch-DeviceServices.md

```diff
@@ -176,6 +176,34 @@ Please refer to the general [Common Configuration documentation](../configuratio
     |---|---|---|
     |MaxEventSize|0|maximum event size in kilobytes sent to Core Data or MessageBus. 0 represents default to system max.|
 
+### URIs for Device Service Files
+
+!!! edgey "EdgeX 3.1"
+    Support for URIs for Devices, Profiles, and Provision Watchers is new in EdgeX 3.1.
+
+When loading  device definitions, device profiles, and provision watchers from a [URI](../general/index.md#uri-for-files), the directory field (ie `DevicesDir`, `ProfilesDir`, `ProvisionWatchersDir`) loads an index file instead of a folder name.
```

Comment:
I am concerned that referencing ../general/index.md#uri-for-files as [URI](../general/index.md#uri-for-files) doesn't jump out to the reader to know that is were to find the URI for Files details. Normally we say something like: See [URI For Files](../general/index.md#uri-for-files) sections for more details.

Probably need to update the previous PR in the same way since It didn't catch it on those.